wp-content/themes/pub/wporg-learn-2024/patterns/archive-lessons-content.php

```diff
@@ -28,6 +28,7 @@
 
 	<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"flex","flexWrap":"nowrap"},"className":"wporg-query-filters"} -->
 	<div class="wp-block-group wporg-query-filters">
+		<!-- wp:wporg/query-filter {"key":"content_type","multiple":false} /-->
```

Comment:
Good question. I added it because if selecting a course or lesson from the search all page, without this filter, you can't go back from the course or lesson archive.

Reply:
Ah, I see, if you do the search first and then filter by type, it moves over to the archive template for those search results.

IMO, people can use the back button to get back to all search results if they filter the post type— it's more confusing to me just landing on `/lessons/` and seeing the content type dropdown, since it brings you to a totally different page (also with different filtering options) if you use it.

Open to other opinions though, cc @ndiego @kathrynwp (not sure who else has input here)

Reply:
I think both options are fine with me. What might ultimately influence my decision is which path the users use most frequently. If most users explore the site from the homepage search, then having a content type filter on the `/lessons/` or `/courses/` pages would provide a better experience IMO.

Alternatively, we could consider showing the filter on the `/lessons/` or `/courses/` pages only when users use the content type filter from the all search results page and are redirected to those pages. If they visit those pages directly, the filter would not be displayed.

I've removed the filter from the lesson and course archive for now. Let me know if we want it back.
b6be6ea

Reply:
> IMO, people can use the back button to get back to all search results if they filter the post type— it's more confusing to me just landing on /lessons/ and seeing the content type dropdown, since it brings you to a totally different page (also with different filtering options) if you use it.

That's a good point. I have no strong opinions though. I found it nice that I could switch over to Courses from Lessons, but I'm not a typical user.